### PR TITLE
Document vitup mock

### PR DESCRIPTION
**Pull request by dkijania**
Created documentation for catalyst backend mock.

### CODE REVIEW

File: book/src/core-testing-doc/api/vitup/mock/Mock.postman_collection.json [hunk 1]
```diff
@@ -0,0 +1,804 @@
...
+					"path": [
+						"api",
+						"control",
+						"command",
+						"available",
+						"true"
+					]
+				},
+				"description": "Controls backend availability. By default it is set to true, and anyone can connect to mock (unless mock is configured with access tokan and client does not have it). This endpoint can alter mock behaviour and make it unvailable for all the call returning always an error code (predefined in error-codes endpoint)"
+			},
```
Comment (kukkok3) [inline book/src/core-testing-doc/api/vitup/mock/Mock.postman_collection.json:40]: Some typos *token *unavailable

File: book/src/core-testing-doc/api/vitup/mock/Mock.postman_collection.json [hunk 2]
```diff
@@ -0,0 +1,804 @@
...
+					"path": [
+						"api",
+						"control",
+						"command",
+						"error-code",
+						"500"
+					]
+				},
+				"description": "Sets return error code when mock is configurred as not available"
+			},
```
Comment (kukkok3) [inline book/src/core-testing-doc/api/vitup/mock/Mock.postman_collection.json:108]: Typo *configured

File: book/src/core-testing-doc/api/vitup/mock/v0.yaml [hunk 3]
```diff
@@ -0,0 +1,1232 @@
...
+- name: files
+- name: health
+- name: reset
+
+paths:
+  /api/control/command/available/true:
+    post:
+      summary: makes mock available to client calls
+      description: Controls backend availability. By default it is set to true, and anyone can connect to mock (unless mock is configured with access tokan and client does not have it). This endpoint can alter mock behaviour and make it available for all the call (contrary to sibling endpoint with ..available/false ) returning always an error code (predefined in error-codes endpoint)
+      operationId: commandAvailableTrue
```
Comment (kukkok3) [inline book/src/core-testing-doc/api/vitup/mock/v0.yaml:24]: Typo *token

File: book/src/core-testing-doc/api/vitup/mock/v0.yaml [hunk 4]
```diff
@@ -0,0 +1,1232 @@
...
+          type: string
+          example: RBj0OfHw5jT87A
+      responses:
+        '200':
+          description: 'Command accepted'
+  /api/control/command/available/false:
+    post:
+      summary: drops all connection attempts from clients
+      description: Controls backend availability. By default it is set to true, and anyone can connect to mock (unless mock is configured with access tokan and client does not have it). This endpoint can alter mock behaviour and make it unavailable for all the call returning always an error code (predefined in error-codes endpoint)
+      operationId: commandAvailableFalse
```
Comment (kukkok3) [inline book/src/core-testing-doc/api/vitup/mock/v0.yaml:41]: typo *token

**Review by kukkok3 — CHANGES_REQUESTED** [review]
LTGM just some typos to fix

**Review by minikin — APPROVED** [review]

**Review by stevenj — APPROVED** [review]

**Pull request merged**